packages/apps-config/src/endpoints/testing.ts

```diff
@@ -192,6 +192,13 @@ export function createTesting (t: TFunction): LinkOption[] {
         'KILT Protocol': 'wss://full-nodes.kilt.io:9944/'
       }
     },
+    {
+      info: 'kylin',
+      text: t('rpc.test.kylin', 'Kylin Testnet', { ns: 'apps-config' }),
+      providers: {
+        kylin: 'wss://rpc.kylin-node.co.uk'
+      }
+    },
```

Comment:
If this is a parachain, it should only be on the next list - this part is for stand-alone chains. (Since the RPCs match here, they are the same)

```suggestion
```

Reply:
Can you clarify what you mean by next list?

Reply:
This one :)

![image](https://user-images.githubusercontent.com/1424473/119185293-e670ee00-ba76-11eb-9c5e-e3b5a27ab82d.png)

Reply:
Got it, so I actually added it. Thanks mate!